### PR TITLE
Made changes so that super admin will be able to search for review managers using their email addresses

### PR DESCRIPTION
**Pull request by mrsasi7994**
Previously Super Admin Can't able to manager Review Managers using their email Addresses. Now made changes so that As a Super Admin, they can search Review Managers using their email.

### CODE REVIEW

**Comment by bkmaibach** [comment]
This code would let unauthenticated users to check if any email is associated with review managers on a jurisdiction. I'm not sure that is a wise thing to allow. Is there some specific reason we are pursuing this feature?

**Review by bkmaibach — CHANGES_REQUESTED** [review]
Jurisdiction search is a public unauthenticated endpoint, and associated review managers is not public data. If this feature is absolutely necessary, please find a way to prevent the public from uncovering private information.

**Review by bkmaibach — CHANGES_REQUESTED** [review]
Please:

- Use role checking methods where appropriate (.super_admin? etc)
- Handle the case of regional review managers
- apply syntax_tree formatting (see README)

File: app/models/jurisdiction.rb [hunk 1]
```diff
@@ -10,10 +10,11 @@ class Jurisdiction < ApplicationRecord
...
 
   include ActionView::Helpers::SanitizeHelper
-  searchkick searchable: %i[name reverse_qualified_name qualified_name],
-             word_start: %i[name reverse_qualified_name qualified_name],
-             text_start: %i[name reverse_qualified_name qualified_name]
+  searchkick searchable: %i[name reverse_qualified_name qualified_name, review_manager_emails],
+             word_start: %i[name reverse_qualified_name qualified_name, review_manager_emails],
+             text_start: %i[name reverse_qualified_name qualified_name, review_manager_emails]
 
+  SEARCH_DATA_FIELDS = %i[qualified_name reverse_qualified_name regional_district_name name type updated_at review_managers_size reviewers_size permit_applications_size user_ids submission_inbox_set_up created_at review_manager_emails]
```
Comment (sunto) [inline app/models/jurisdiction.rb:17]: I would leave out updated_at and created_at

**Review by sunto — APPROVED** [review]

**Review by guri-dev — APPROVED** [review]

**Review by bkmaibach — APPROVED** [review]
Nicely done

File: app/models/jurisdiction.rb [hunk 2]
```diff
@@ -130,7 +171,9 @@ def search_data
       permit_applications_size: permit_applications_size,
       user_ids: users.pluck(:id),
       submission_inbox_set_up: submission_inbox_set_up,
-      created_at: created_at
+      created_at: created_at,
+      review_manager_emails: review_manager_emails,
+      regional_review_manager_emails: regional_review_manager_emails
```
Comment (bkmaibach) [inline app/models/jurisdiction.rb:176]: This might be better if they were the same field, like manager_emails or something

**Review by bkmaibach — CHANGES_REQUESTED** [review]

**Review by bkmaibach — APPROVED** [review]

**Pull request merged**